common/djangoapps/student/models.py

```diff
@@ -1382,6 +1386,22 @@ def refundable(self):
         else:
             return True
 
+    def _refund_window_end_date(self):
+        """ Calculate and return the refund window end date. """
+        try:
+            attribute = self.attributes.get(namespace='order', name='order_number')  # pylint: disable=no-member
+        except ObjectDoesNotExist:
+            return None
+
+        order_number = attribute.value
+        order = ecommerce_api_client(self.user).orders(order_number).get()
+        refund_window_start_date = max(
+            datetime.strptime(order.get('date_placed'), ECOMMERCE_DATE_FORMAT),
+            self.course_overview.start.replace(tzinfo=None)
```

Comment:
I remember now.  This was done in an attempt to make the two dates comparable. Right now you get the following error when you don't make `course_overview.start` naive 

`TypeError: can't compare offset-naive and offset-aware datetimes`